packages/node-build-scripts/generate-icons-source

```diff
@@ -111,13 +112,13 @@ function exportIconConsts(valueGetter) {
 async function buildPathsObject(objectName, size) {
     return Promise.all(
         ICONS_METADATA.map(async icon => {
-            const filepath = path.resolve(__dirname, `../../resources/icons/${size}px/${icon.className}.svg`);
+            const filepath = path.resolve(__dirname, `../../resources/icons/${size}px/${toLongName(icon)}.svg`);
```

Comment:
can we batch rename the SVG files too? no need for pt- in the filename

Reply:
oof, sure